modules/veteran_confirmation/catalog-info.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: backstage.io/v1alpha1
+kind: API
+metadata:
+  name: veteran-confirmation
+  description: Veteran Confirmation APIs for VA Lighthouse program
+  title: Veteran Confirmation API
+  tags:
+  - lighthouse
+  - external
+  links:
+  - url: https://developer.va.gov/explore/verification/docs/veteran_confirmation?version=current
+    title: Public Docs
+    icon: web
+spec:
+  type: openapi
+  lifecycle: production
+  owner: lighthouse-pivot
+  definition:
+    $text: https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/veteran_confirmation/VETERAN_CONFIRMATION.yml
```

Comment:
Confirmed based on Dev Hub docs this could be a yml or json
https://department-of-veterans-affairs.github.io/lighthouse-developer-portal/catalog-entry/#spectype-required_2

Since this is for an internal portal, holding off on recommending the DocServer routed OpenAPI specification used for the Developer Portal.